Automattic/jetpack
# Update boost score bar to use reusable component

Replaces existing `ScoreBar.svelte` component with the package React component. Also removes unused code

## Proposed changes:
 * Remove `ScoreBar.svelte`
 * Use React Score Bar package component instead
 * Remove unused SVGs
 * Remove unused CSS

### Other information:


## Jetpack product discussion

P2: p1HpG7-k9o-p2

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Checkout these changes - `git switch update/boost-score-bar-to-use-reusable-component`
2. Start up your local environment
3. Run `jetpack install plugins/boost && jetpack build plugins/boost`
*Alternatively to steps 2 and 3, you can use the Jetpack Beta plugin on a jurassic ninja site
4. If you haven't already, install the Jetpack Boost plugin on your site
5. Go to `/wp-admin/admin.php?page=jetpack-boost` and make sure the score bars look and behave as they did before
![image](https://github.com/Automattic/jetpack/assets/65001528/01a2a9aa-8f1c-41cb-a392-8f0924d024ee)
